runtime/src/accounts.rs

```diff
@@ -229,18 +230,35 @@ impl Accounts {
     }
 
     /// If feature `cap_transaction_accounts_data_size` is active, total accounts data a
-    /// transaction can load is limited to 64MiB to not break anyone in Mainnet-beta today.
-    /// (It will be set by compute_budget instruction in the future to more reasonable level).
+    /// transaction can load is limited to
+    ///   if `set_tx_loaded_accounts_data_size` instruction is not activated or not used, then
+    ///     default value of 64MiB to not break anyone in Mainnet-beta today
+    ///   else
+    ///     user requested loaded accounts size.
+    ///     Note, requesting zero bytes will result transaction error
     fn get_requested_loaded_accounts_data_size_limit(
+        tx: &SanitizedTransaction,
```

Comment:
Mmm, didn't look it that way, just thought `get_requested_loaded_accounts_data_size_limit` is an action on `tx`, details of finding data_size_limit of `tx` can be encapsulated within _this_ function. 

Edit: relates to other comment, if `tx` caches result of `process_instruction()`, then it'd also make sense to pass in `tx` instead of iter of instructions.